docs/lib/graphqlapi/fragments/ios/advanced-workflows.md

```diff
@@ -0,0 +1,225 @@
+This section describes different use cases for constructing your own custom GraphQL requests and how to approach it. You may want to construct your own GraphQL request if you want to
+- retrieve only a subset of the data to reduce data transfer
+- retrieve nested objects at a depth that you choose
+- combine multiple operations into a single request
+- decode the GraphQL response to a format of your choice
```

Comment:
Since Amplify is an opinionated framework, we should avoid giving customers a "choice" of response format.    I do get that you need to explain what `responseType` and `decodePath` should be, but I would just remove this bullet point.

Reply:
removing